styleguide/Repositories/PromoRepository.php

```diff
@@ -111,7 +109,21 @@ public function getRequestData(array $data)
         ];
 
         // Only pull hero promos if they match the page ids that are specificed
-        $styleguide_hero = !empty($hero_page_ids[$data['page']['id']]) ? $hero_page_ids[$data['page']['id']] : null;
+        $styleguide_hero = [
+            'data' => !empty($hero_page_ids[$data['page']['id']]) ? $hero_page_ids[$data['page']['id']] : null,
+            'component' => [
+                //'option' => '',
+            ]
+        ];
+
+        // TODO This is weak
```

Comment:
What does this mean weak? What edge cases should we also consider to make it stronger?